cmd/crc/cmd/status.go

```diff
@@ -34,23 +45,24 @@ Cache Directory: {{.CacheDir}}
 `
 
 type Status struct {
-	CrcStatus       string
-	OpenShiftStatus string
-	DiskUsage       string
-	CacheUsage      string
-	CacheDir        string
+	CrcStatus       string `json:"crcStatus"`
```

Comment:
We do not care much about the actual markup, but rather about providing the needed information. This would be more important is we would ever rename something.

Reply:
Markup is better if it's consistent with other products. OpenShift and Kubernetes use lower camel case. CRC should do the same.